app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -254,7 +258,8 @@ class SuggestedEditsTasksFragment : Fragment() {
 
     private fun setUserStatsViewsAndTooltips() {
         binding.contributionsStatsView.setImageDrawable(R.drawable.ic_mode_edit_white_24dp)
-        binding.contributionsStatsView.tooltipText = getString(R.string.suggested_edits_contributions_stat_tooltip)
+        binding.contributionsStatsView.tooltipText = if (ContributionsDashboardHelper.contributionsDashboardEnabled)
+            getString(R.string.suggested_edits_edit_stat_tooltip) else getString(R.string.suggested_edits_contributions_stat_tooltip)
```

Comment:
Since we are not going to show tooltip for the experiment, it is okay to skip the change here.